Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -620,11 +619,11 @@ UpdateFspConfig (
         for (Index = 0; Index < PCH_MAX_HDA_SSP_LINK_NUM; Index++) {
           Fspmcfg->PchHdaAudioLinkSspEnable[Index]      = 0;
         }
-        if (!FeaturePcdGet (PcdPreOsCheckerEnabled) && !PchIsSciSupported ()) {
+        if (PchIsSciSupported () != 1) {

```

Comment:
PchIsSciSupported() has been called many times.  Recommend to use a variable to store the result so as to avoid duplicated call again and again.   Current code is not efficient since each call will trigger HW PCI CFG register access cycle.

Reply:
Can you address this in your patch ?

Reply:
Sure I’m working on it

Reply:
Updated the patch